app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,93 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.job_id
+
+    send_spotify_request
```

Comment:
It'd improve readability and testability to move this stuff out into a separate object. This might be a good test-driven design point.